docs/containerd-2.0.md

```diff
@@ -0,0 +1,138 @@
+
+This release adds the `PluginInfo` API definition to the introspection service. (`/containerd.services.introspection.v1.Introspection/PluginInfo`)
+
+For example, the new `PluginInfo()` call can be used to inspect the version, features, and annotations of the runtime plugin.
+
+```bash
+$ ctr plugins inspect-runtime --runtime=io.containerd.runc.v2 --runc-binary=runc
+{
+    "Name": "io.containerd.runc.v2",
+    "Version": {
+        "Version": "v2.0.0-rc.X-XX-gXXXXXXXXX.m",
+        "Revision": "v2.0.0-rc.X-XX-gXXXXXXXXX.m"
+    },
+    "Options": {
+        "binary_name": "runc"
+    },
+    "Features": {
+        "ociVersionMin": "1.0.0",
+        "ociVersionMax": "1.2.0",
+        ...,
+    },
+    "Annotations": null
+}
+```
+
+### OTEL environment variable configuration support for containerd's built-in tracing plugin
+
+This release adds support for the standard environment variables defined in the OpenTelemetry [specification](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/) to the built-in containerd tracing plugin.
+
+Implementation note: by default OTEL connects to `https://localhost:4318` if no endpoint is set. In contrast, the containerd tracing OLTP plugin is disabled if no endpoint is configured.
+
+i.e. both `OTEL_SDK_DISABLED` and one of either `OTEL_EXPORTER_OTLP_ENDPOINT` or `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` must be set in the containerd environment for the tracing plugin to be enabled.
+
+### Intel ISA-L's igzip support
+
+Intel ISA-L's igzip support has been added to the containerd client. If found, the containerd client uses igzip for gzip decompression. Benchmarks have shown igzip to outperform both Go's built-in gzip and external pigz implementations.
+
+### Deprecation warnings can now be discovered via the Introspection API
+
+This release adds deprecations warnings to the `ServerResponse` for the introspection service. (`/containerd.services.introspection.v1.Introspection/Server`)
+
+To ease containerd 2.0 transition, the community has moved to backport this feature to containerd 1.6 and 1.7 release branches to give users another tool to identify the sticking points for migrating containerd versions.
+
+Administrators whose workloads are running on containerd versions >= 1.6.25, >= 1.7.9 can query their containerd server to receive  warnings for features marked for deprecation and are in the critical path. With the `ctr` client, users can run `ctr deprecations list`.
+
+## What's breaking
+
+### Docker Schema 1 image support has been disabled
+
+Pulling Docker Schema 1 (`application/vnd.docker.distribution.manifest.v1+json`) images has been disabled by default in containerd 2.0. Users can re-enable previous behavior by setting `CONTAINERD_ENABLE_DEPRECATED_PULL_SCHEMA_1_IMAGE=1`; however, users are **recommended** to begin migration to Docker Schema 2 or OCI images. Support for Docker Schema 1 images will be fully removed in containerd 2.1.
```

Comment:
Maybe add how to migrate images (e.g., just rebuild/push them with the latest Docker or whatever)